firmware_configurations/klipper/clee/fysetc_s6/v0-fysetc_s6_v2.1.cfg

```diff
@@ -0,0 +1,323 @@
+
+[extruder]
+step_pin: MOT3_STEP
+dir_pin: MOT3_DIR                                                   # Add ! if moving opposite direction
+enable_pin: !MOT3_DIR
+#full_steps_per_rotation: 200                                       # Set to 200 for LDO 1.8° stepper motor, and set to 400 for OMC(StepperOnline) 0.9° stepper motor
+rotation_distance: 22.23                                            # See calibrating rotation_distance on extruders doc
+gear_ratio: 50:10                                                   # For Mini Afterburner
+microsteps: 32
+nozzle_diameter: 0.400
+filament_diameter: 1.750
+heater_pin: HE0
+##  Validate the following thermistor type to make sure it is correct
+##  See https://www.klipper3d.org/Config_Reference.html#common-thermistors for additional options
+#sensor_type: Trianglelab NTC100K B3950                             # Adjust for your hotend thermistor.
+sensor_pin: T0
+control: pid                                                        # Do PID calibration after initial checks
+pid_Kp: 28.182
+pid_Ki: 1.978
+pid_Kd: 100.397
+min_temp: 0
+max_temp: 270
+min_extrude_temp: 170
+max_extrude_only_distance: 150
+max_extrude_cross_section: 0.8
+pressure_advance: 0.0                                               # See tuning pressure advance doc
+pressure_advance_smooth_time: 0.040
+
+[tmc2209 extruder]
+uart_pin: MOT3_CS
+interpolate: False
+## For OMC (StepperOnline) 14HR07-1004VRN 1A 0.9°
+#run_current: 0.5	# for OMC 14HR07-1004VRN rated at 1A
+## For LDO LDO 36STH17-1004AHG 1A 1.8° 
+#run_current: 0.3	# for LDO 36STH17-1004AHG
+stealthchop_threshold: 0                                            # Set to 0 for spreadcycle, avoid using stealthchop on extruder
+
+#[extruder1]
+#step_pin: MOT4_STEP
+#dir_pin: !MOT4_DIR
+#enable_pin: !MOT4_EN
+#heater_pin: HE1
+#sensor_pin: T1
+
+#[tmc2209 extruder1]
+#uart_pin: MOT4_CS
+## For OMC (StepperOnline) 14HR07-1004VRN 1A 0.9°
+#run_current: 0.5	# for OMC 14HR07-1004VRN rated at 1A
+## For LDO LDO 36STH17-1004AHG 1A 1.8° 
+#run_current: 0.3	# for LDO 36STH17-1004AHG
+#stealthchop_threshold: 0
+
+#[extruder2]
+#step_pin: MOT5_STEP
+#dir_pin: !MOT5_DIR
+#enable_pin: !MOT5_EN
+#heater_pin: HE2
+#sensor_pin: T2
+
+#[tmc2209 extruder2]
+#uart_pin: MOT5_CS
+## For OMC (StepperOnline) 14HR07-1004VRN 1A 0.9°
+#run_current: 0.5	# for OMC 14HR07-1004VRN rated at 1A
+## For LDO LDO 36STH17-1004AHG 1A 1.8° 
+#run_current: 0.3	# for LDO 36STH17-1004AHG
+#stealthchop_threshold: 0
+
+#####################################################################
+#   Bed Heater
+#####################################################################
+
+[heater_bed]
+heater_pin: BED_OUT
+##  Validate the following thermistor type to make sure it is correct
+##  See https://www.klipper3d.org/Config_Reference.html#common-thermistors for additional options
+#sensor_type: NTC 100K MGB18-104F39050L32                           # For Keenovo, verify yours
```

Comment:
according to Eddie, keenovo says we should be using Generic 3950 for their heaters. Have you heard differently?

Reply:
I started with the SKR Pico V0.1 config file as my template: <https://github.com/VoronDesign/Voron-0/blob/Voron0.1/Firmware/skr-pico-v1.0.cfg> which is where that came from.

Reply:
Updated to `Generic 3950`.